src/test/java/org/minions/devfund/noemiguzman/moviedatabase/MovieDatabaseTest.java

```diff
@@ -0,0 +1,246 @@
+        assertTrue(movieList.equals(moviedb3.getMovieList()));
+    }
+
+    /**
+     *  Method to test add movie.
+     */
+    @Test
+    public void testAddMovieWithActor() {
+        movieList.add(movie2);
+
+        String[] actors2;
+        actors2 = new String[] {actoreAffleck, actoreWest, actoreBale};
+
+        String[] newActors2 = actors2.clone();
+        assertArrayEquals(newActors2, actors2);
+        assertFalse(newActors2 == actors2);
+        moviedb3.addMovie(movieBatman, newActors2);
+        assertTrue(movieList.equals(moviedb3.getMovieList()));
+    }
+    /**
+     *  Method to test add rating.
+     */
+    @Test
+    public void testAddRating() {
+        final Double expectedRating = 7.8;
+        String[] actors = new String[] {actoreAffleck, actoreWest};
+        moviedb2.addMovie(movieBatman, actors);
+        moviedb2.addRating(movieBatman, expectedRating);
+        assertEquals(expectedRating, moviedb2.getMovieList().get(0).getRating(), 0);
+    }
+
+    /**
+     *  Method to test update rating.
+     */
+    @Test
+    public void testUpdateRating() {
+        String[] actors = new String[] {actoreAffleck, actoreWest};
+        final Double expected1 =  25.10;
+        final Double expectedFinal = 32.10;
+        moviedb2.addMovie(movieBatman, actors);
+        moviedb2.addRating(movieBatman, expected1);
+        moviedb2.updateRating(movieBatman, expectedFinal);
+        assertEquals(expectedFinal, moviedb2.getMovieList().get(0).getRating(), 0);
+    }
+
+    /**
+     * Method to test get best actor.
+     */
+    @Test
+    public void testGetBestActor() {
+
+        String[] actors1 = new String[] {actoreAffleck, actoreWest};
+        String expectedBestActor = actoreAffleck;
+        final Double expectedRating1 = 5.5;
+        moviedb1.addMovie(movieGlass, actors1);
+        moviedb1.addRating(movieGlass, expectedRating1);
+
+        String[] actors2;
+        actors2 = new String[] {actoreWest, actoreAffleck, actoreBale};
+        final Double expectedRating2 = 6.5;
+        moviedb1.addMovie(movieBatman, actors2);
+        moviedb1.addRating(movieBatman, expectedRating2);
+
+        String[] actors3;
+        actors3 = new String[] {actoreWest, actoreBale};
+        final Double expectedRating3 = 7.5;
+        moviedb1.addMovie("It", actors3);
+        moviedb1.addRating("It", expectedRating3);
+
+        // actor1 movies
+        movie1.addActor(actor2);
+        movie2.addActor(actor1);
+        movie2.addActor(actor3);
+        movie3.addActor(actor2);
+
+        movieList1.add(movie2);
+
+        // actor2 movies
+        movieList2.add(movie1);
+        movieList2.add(movie3);
+
+        // actor3 movies
+        movieList3.add(movie2);
+
+        // Test movie ratings
+        movie1.setRating(expectedRating1);
+        movie2.setRating(expectedRating2);
+        movie3.setRating(expectedRating3);
+        System.out.println(expectedBestActor);
+        System.out.println(moviedb1.getBestActor());
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Replace this use of System.out or System.err by a logger. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS106)